src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -28,10 +29,17 @@ public void AddTrace(string trace)
 
         ConcurrentQueue<string> traceQueue = new ConcurrentQueue<string>();
 
+        internal Dictionary<string, LogLevel> LogLevels = new Dictionary<string, LogLevel>();
+
+        public void SetLogLevel(string logger, LogLevel level)
```

Comment:
This was the main purpose I started this PR. If I switch to `debug` across the board, I get a lot of stuff that is not important when I want to get some specific information.